tools/pd-ctl/pdctl/ctl_test.go

```diff
@@ -0,0 +1,73 @@
+package pdctl
+
+import (
+	"github.com/spf13/cobra"
+	"testing"
+)
+
+func newACommand() *cobra.Command {
+	a := &cobra.Command{
+		Use:   "testa",
+		Short: "test A command",
+	}
+	return a
+}
+
+func newBCommand() *cobra.Command {
+	a := &cobra.Command{
+		Use:   "testb",
+		Short: "test b command",
+	}
+	return a
+}
+func newDEFCommand() *cobra.Command {
+	a := &cobra.Command{
+		Use:   "testdef",
+		Short: "test def command",
+	}
+	return a
+}
+
+func newCCommand() *cobra.Command {
+	a := &cobra.Command{
+		Use:   "testc",
+		Short: "test c command",
+	}
+	return a
+}
```

Comment:
Can we simplify these methods into one method which receive the parameters of `Use` and `Short`?

Reply:
recieve use

Reply:
I just simulate the use of cobra in pdctl